src/main/java/seedu/storemando/logic/commands/ReminderCommand.java

```diff
@@ -0,0 +1,41 @@
+package seedu.storemando.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.storemando.commons.core.Messages;
+import seedu.storemando.model.Model;
+import seedu.storemando.model.item.ItemExpiringPredicate;
+
+/**
+ * Finds and lists all items in storemando whose item's expiry date is within a certain days from today.
+ */
+public class ReminderCommand extends Command {
+
+    public static final String COMMAND_WORD = "reminder";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Filters all items whose expiry date is within "
+        + "certain days and displays them as a list with index numbers.\n"
```

Comment:
Would it be better if it explicitly states "user-specified number of days from the current date..."?

Reply:
Agree with Fazil!

Reply:
All right noted! Will change accordingly.